From: Henry Elderman <henry.elderman.edu@gmail.com>
Subject: mm/execmem: fix fallback_end description in kernel-doc
Date: Fri, 7 Aug 2026 11:19:58 +0200

The kernel-doc for struct execmem_range incorrectly describes @fallback_end
as "start". Correct it to "end".

Link: https://lore.kernel.org/20260807091958.4735-1-henry.elderman.edu+linux@gmail.com
Signed-off-by: Henry Elderman <henry.elderman.edu+linux@gmail.com>
Cc: Mike Rapoport <rppt@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/execmem.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/execmem.h~mm-execmem-fix-fallback_end-description-in-kernel-doc
+++ a/include/linux/execmem.h
@@ -89,7 +89,7 @@ static inline int execmem_restore_rox(vo
  * @end:	address space end (inclusive)
  * @fallback_start: start of the secondary address space range for fallback
  *                  allocations on architectures that require it
- * @fallback_end:   start of the secondary address space (inclusive)
+ * @fallback_end:   end of the secondary address space (inclusive)
  * @pgprot:	permissions for memory in this address space
  * @alignment:	alignment required for text allocations
  * @flags:	options for memory allocations for this range
_
